controller/server/main.go

```diff
@@ -43,6 +47,8 @@ func init() {
 		defaultMeshnetManifestDir = filepath.Join(home, "kne", "manifests", "meshnet", "base")
 		defaultMetallbManifestDir = filepath.Join(home, "kne", "manifests", "metallb")
 	}
+	// flag.IntVar(&port, "port", 50051, "Controller server port")
+	flag.StringVar(&kubecfg, "kubecfg", defaultKubeCfg, "Kube config")
```

Comment:
the kubecfg flag can be removed. it makes sense for the CLI, but does not make sense at server creation time. instead the user passes the kubecfg field in the request for topo creation as I commented down below